zerver/tests/test_subs.py

```diff
@@ -1022,6 +1022,39 @@ def test_api_calls(self) -> None:
         result = self.client_delete('/json/default_stream_groups/{}'.format(group_id))
         self.assert_json_error(result, "Default stream group with id '{}' does not exist.".format(group_id))
 
+    def test_invalid_default_stream_group_name(self) -> None:
+        self.login(self.example_email("iago"))
+        user_profile = self.example_user('iago')
+        realm = user_profile.realm
+
+        stream_names = ["stream1", "stream2", "stream3"]
+        group_name = ""
```

Comment:
I think it'd be more readable if you inlined this variable where it's used, rather than changing it for each test.